MIP/MIP-60/README.md

```diff
@@ -0,0 +1,234 @@
+- **Canonical Bridges** is a term typically used to refer to L1 to L2 bridges, and have a similar meaning to Native Bridges. They typically take a longer period of time to bridge assets than liquidity networks as they require a time period for finality to be established, such as the challenge period for optimistic
+rollups. 
+- **Validating Bridges** are bridges between Ethereum and a rollup, where the bridge contract verifies the state updates proposed by the offchain system.
+- **Validator or Oracle-based Bridges** rely on a set of external validators or oracles to validate cross-chain transfers.
+
+> [!NOTE]
+> The term **Native Bridge** is used to refer to a bridge that transfers the native ERC-20 `$L1MOVE` token from a base chain (L1) to the `$L2MOVE` token on a different chain (L2).
+
+**Liquidity Networks** [[5]][REF5]
+
+The term of Liquidity Networks is associated with 
+
+- **Continuous Liquidity Pools** The majority of Liquidity Networks consist of two liquidity pools of assets - one on the source and the other on the destination chain. In this structure, an intermediary token can act as an exchange medium allowing accurate amounts of the final token to be transferred to the user and to rebalance the pools once a transaction has occurred, e.g. RUNE in Thorchain, or stablecoins such as USDC in Stargate. 
+
+In our setting the intermediary IS the final token, and this could be interpreted as a **lock/unlock** bridge.
+
+- **Continuous Liquidity Pools (CLP) with AMMs** are a subset of Continuous Liquidity Pools that use Automated Market Makers (AMMs) in series. [MIP-40](https://github.com/movementlabsxyz/MIP/pull/40) has proposed such a design.
+
+**Types of bridge mechanisms**
+
+There are three types of crosschain bridges. They differ in the way they handle the native assets (source chain) and the representation of the assets (wrapped asset) on the target chain.
+
+1. **burn/mint**: the asset is burned on the source chain and an asset is minted on the target chain.
+_Supply: The bridge operator is in control of creating supply on both chains._
+1. **lock/mint**: the native asset (minted e.g. in an ERC20 contract) is locked on the source chain and a wrapped asset is minted on the target chain (which is L2 in our case).
+_Supply: The bridge operator is only in control of minting supply on the target chain (L2). On L1 the bridge operator holds a token pool into/from which tokens get locked/unlocked._
+1. **lock/unlock**: although difficult to classify in literature, this variant naturally completes the above two types. It may be classified as a subcategory of a swap, a **liquidity network** (see above), or it could be interpreted as a trusted [Automated Market Maker (AMM) with a constant sum invariant](https://medium.com/@0xmirai77/curve-v1-the-stableswap-invariant-f87ad7641aa0). The asset is locked on the source chain and unlocked on the target chain. 
```

Comment:
I would say any **mint** with with a cap on how much can be minted is effectively an **unlock.**